### DISCUSSION TITLE
Cancellazione segnalazioni dopo 90 giorni

### DISCUSSION THREAD

**Discussion by cienne1** [globaleaks/globaleaks-whistleblowing-software#3693] [General]
Il sistema ha inviato una email di promemoria (relativa a segnalazioni di prova che avevamo inserito il 14 luglio scorso) con cui mi avvisa che alcune segnalazioni saranno cancellate il 13 ottobre, a 90 giorni dall'inserimento della segnalazione.
Se questo succedesse con le segnalazioni vere sarebbe un problema perchè l'istruttoria potrebbe non essere finita. C'è modo di evitare che le segnalazioni vengano cancellate?
![image](https://github.com/globaleaks/GlobaLeaks/assets/141617580/9d6c540e-76de-4842-bf6a-80e6db71cb23)

**Comment by cienne1** [comment]
Qualcuno ha avuto modo di verificare (anche in test) che cosa succede alle segnalazioni dopo 90 giorni?

**Comment by evilaliv3** [comment]
Buonasera @cienne1 

GlobaLeaks di default importa una cancellazione automatica delle segnalazioni a 90 giorni.

Questa configurazione del canale di segnalazione di default può essere variata dagli amministrazioni del sistema secondo preferenza.

By design e by default in italia tipicamente questa tempistica viene portata ad 1 massimo 2 anni.

Questa configurazione viene poi combinata dando il permesso ai riceventi di variare anche manualmente la scadenza delle segnalazioni secondo necessità
